.github/workflows/cmake.yml

```diff
@@ -127,7 +127,7 @@ jobs:
 
     - name: Test
       working-directory: ${{github.workspace}}/build
-      run: ctest -C ${{matrix.build_type}} --output-on-failure -L "umf|loader|validation|tracing|unit|urtrace"
+      run: ctest -C ${{matrix.build_type}} --output-on-failure
```

Comment:
This doesn't seem like a change we should make in this PR.

Reply:
Shall I expand the scope of this MR (via adding it to the title/description) or update the testing in a separate MR?

Reply:
*If* we make this change, I think it should be a separate PR. This makes this not NFC, for testing coverage anyway.